### ISSUE TITLE
508-defect-3 [SCREENREADER] Headings are improperly nested on the main page

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21480]
Label: 20133

    - [508-defect-3]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3

Issue Title: Main page: headings improperly nested
 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela 

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect headings to be nested so I can easily move between major headings.
<h2>Details</h2>
The level three headings "Manage your health online" and “Get updates from VA Sheridan Health Care” are level 3 headings. Since they represent changes in the subject of discussion within the page, they should be level 2 headings.

Acceptance Criteria

Environment

Steps to Recreate

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs
